Cargo.toml

```diff
@@ -14,15 +14,8 @@ include = ["src/**/*", "Cargo.toml", "CHANGELOG.md", "LICENSE", "README.md", "et
 
 [dependencies]
 plain = "0.2.3"
-
-[dependencies.log]
-version = "0.4"
-default-features = false
-optional = true
-
-[dependencies.scroll]
-version = "0.9"
-default_features = false
+log = { version = "0.4", default-features = false, optional = true }
+scroll = { version = "0.9", default-features = false }
```

Comment:
Why? I'm not convinced that this is objectively better.

Reply:
I don't think one is better than another but the new one is shorter.
I'd revert if you prefer.

Reply:
Yes, please drop that commit.

Reply:
Done.